src/gtest/test_mnode_governance.cpp

```diff
@@ -41,13 +41,13 @@ TEST(mnode_governance, TicketProcessing) {
 
     //test of AddTicket logic
     std::string note1("ticket1");
-    CGovernanceTicket ticket1(scriptPubKey, 3125000000, note1, 0+1001); //31250 - need 100 blocks to pay
+    CGovernanceTicket ticket1(scriptPubKey, 3125000000, note1, 0+1001); //31250 - need 50 blocks to pay
```

Comment:
BTW, since c++14 we can use digits separator ' - it makes huge numbers much more readable... For example,
3'125'000'000

Reply:
This is from commits on master branch after rebasing. Not belong to this PR.